libs/editor/example/src/main/java/org/wordpress/android/editor/example/EditorExampleActivity.java

```diff
@@ -57,27 +61,58 @@
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         if (getIntent().getIntExtra(EDITOR_PARAM, USE_NEW_EDITOR) == USE_NEW_EDITOR) {
-            ToastUtils.showToast(this, R.string.starting_new_editor);
-            setContentView(R.layout.activity_new_editor);
+            ToastUtils.showToast(this, R.string.starting_aztec_editor);
+            setContentView(R.layout.activity_aztec_editor);
         } else {
-            ToastUtils.showToast(this, R.string.starting_legacy_editor);
-            setContentView(R.layout.activity_legacy_editor);
+            Fragment fragment = GutenbergEditorFragment.newInstance("", "", true, "us-en");
+            ToastUtils.showToast(this, R.string.starting_gutenberg_editor);
+            setContentView(R.layout.activity_gutenberg_editor);
+            FragmentTransaction ft = getSupportFragmentManager().beginTransaction();
+            ft.replace(R.id.postEditor, fragment);
+            ft.commit();
         }
 
         mFailedUploads = new HashMap<>();
     }
 
     @Override
-    public void onAttachFragment(Fragment fragment) {
+    public void onAttachFragment(androidx.fragment.app.Fragment fragment) {
         super.onAttachFragment(fragment);
         if (fragment instanceof EditorFragmentAbstract) {
             mEditorFragment = (EditorFragmentAbstract) fragment;
         }
+        if (fragment instanceof AztecEditorFragment) {
+            // Fake loaders
+            ((AztecEditorFragment) mEditorFragment).setAztecImageLoader(new ImageGetter() {
+                @Override
+                public void loadImage(String source, Callbacks callbacks, int maxWidth) {
+
+                }
+
+                @Override
+                public void loadImage(String source, Callbacks callbacks, int maxWidth,
+                                      int minWidth) {
+
+                }
+            });
+            ((AztecEditorFragment) mEditorFragment).setAztecVideoLoader(new VideoThumbnailGetter() {
+                @Override
+                public void loadVideoThumbnail(String source, Callbacks callbacks, int maxWidth) {
+
+                }
+
+                @Override
+                public void loadVideoThumbnail(String source, Callbacks callbacks, int maxWidth,
+                                               int minWidth) {
+
+                }
+            });
+        }
     }
 
     @Override
     public void onBackPressed() {
-        Fragment fragment =  getFragmentManager()
+        androidx.fragment.app.Fragment fragment =  getSupportFragmentManager()
```

Comment:
I think that there is no need for a full package path as we already have import on the top of the file?

Reply:
Yep, right, I'll update this and the previous one and do a quick check to see if there are other occurences.

Reply:
Fixed in 0a7e0b6b6d (I couldn't find any other occurences)